tests/system/test_sqlalchemy_bigquery.py

```diff
@@ -283,56 +291,73 @@ def test_reflect_select(table, table_using_test_dataset):
         # when a destination table is specified, even though no
         # destination table is specified. When this test was written,
         # `use_labels` was forced by the dialect.
-        rows = table.select(use_labels=True).execute().fetchall()
-        assert len(rows) == 1000
+        with engine_using_test_dataset.connect() as conn:
```

Comment:
The above comment seems like it is no longer relevant. IF so, then please delete.